main/Dom_o_Zielonych_Progach/lemata.tex

```diff
@@ -0,0 +1,61 @@
+\tytul{Łemata (Rozmyślania nad wodospadem)}{sł. muz. Wojtek Neron Warchoł}{Dom o Zielonych Progach}
+\begin{text}
+    Pamiętam tylko tabun chmur się rozwinął\\    
+    I cichy wiatr wiejący ku połoninom\\
+	I twardy jak kamień plecak pod moją głową\\
```

Comment:
Taby zamiast spacji (i powtarza się to w kilku miejscach w tym pliku)

Reply:
Nie rozumiem tego, ale to chyba ma cos wspolnego z tym, ze oryginalnie przy przepisywaniu tego tekstu nie mialem ustawionego taba jako tych 4 spacji. Czy teraz, po wyrownaniu tego tekstu jest ok czy dalej nie bangla? jeżeli nie bangla, to daj znac co trzeba zrobic

Reply:
Są trochę wymieszane nadal taby i spacje, jak sobie popatrzysz jakie znaki są na początku każdej linii.

Tak jak pisałem w pull requeście obok: w notepadzie++ zaznaczasz całą sekcję, shift+tab usuwa wcięcia a później sam TAB je dodaje, już ze spacjami jeśli masz dobrze ustawionego :)

Reply:
Ok, zrobiłem tak, jak pisałeś ale w tym miejscu - na githubie. Czy może chodziło Ci o to, ze trzeba ten tekst wyciąć, poprawić go w notepadzie ++ i ponownie wkleić?